### ISSUE TITLE
Cocoapod version not the current master version

### ISSUE THREAD

**Issue by gabehoffman** [okta/okta-sdk-appauth-ios#109]
I'm not sure what's happening with `signOutFromOkta, but it's clear there there are some versioning issues. When I updated to the latest pod and try to use the method `signOutFromOkta` as documented in the Readme file, it's not found. Then I looked at the example project and it uses a method named `signOutOfOkta`. When I look at it's pod files, while supposedly being the same version they quite different. For example:

My Pods - 
```
Using AppAuth (0.91.0)
Installing HydraAsync (1.0.2)
Installing OktaAuth 1.0.1 (was 0.3.0 and source changed to `https://github.com/CocoaPods/Specs.git` from `https://github.com/cocoapods/specs.git`)
Installing OktaJWT (1.0.0)
Installing Vinculum (0.2.0)
```

OktaAppAuth.swift lines 29-42
```
// Token manager
public var tokens: OktaTokenManager?

public func login(_ username: String, password: String) -> Login {
    // Authenticate via Resource Owner Password Grant
    return Login(forUsername: username, forPassword: password)
}

public func login() -> Login {
    // Authenticate via authorization code flow
    return Login()
}

public func isAuthenticated() -> Bool {
```

Example Project Pods - 
```
Installing HydraAsync (1.2.1)
Installing OktaAppAuth (1.0.0.beta-okta)
Installing OktaAuth (1.0.1)
```

OktaAppAuth.swift lines 29-42
```
public var tokens: OktaTokenManager?

public func login() -> Login {
    // Authenticate via authorization code flow
    return Login()
}

public func signOutOfOkta() -> Logout {
    // End the Okta session
    return Logout()
}

public func isAuthenticated() -> Bool {
```

So, it looks like there is signOut in the example, and looking through the source in the master repo it's there as well, but it's not the same code as the Readme, or the publicly available pod. And the example has obviously different dependencies. (Updated HydraAsync, OktaAppAuth, no Vinculum). Maybe the cocoapod isn't pointing to the latest code, I'm not sure.

**Comment by gabehoffman** [comment]
One option to get the current master version is to force Cocoapods to look at this repo:
```
pod 'OktaAuth', :git => 'git@github.com:okta/okta-sdk-appauth-ios.git'
```

**Comment by nbarbettini** [comment]
> Maybe the cocoapod isn't pointing to the latest code, I'm not sure.

That's the case. This was my fault, sorry - I jumped the gun in merging https://github.com/okta/okta-sdk-appauth-ios/pull/106. We haven't released [2.0.0](https://github.com/okta/okta-sdk-appauth-ios/milestone/4) yet, which includes the `signOutFromOkta` method and the other things you're seeing in the code (but not seeing in Cocoapods).

I'll be more careful of that in the future. We plan to release 2.0.0 very soon; I'll leave this open until we do.

**Comment by nbarbettini** [comment]
2.0.0 is now published to Cocoapods: https://github.com/okta/okta-sdk-appauth-ios/releases/tag/2.0.0

Please open a new issue if you are having new issues after you upgrade to 2.0.0. 👍
